projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -9,97 +9,32 @@
 @use '../../image/icons.clarity';
 @use '../../image/images.clarity';
 @use '../../utils/mixins';
-@use '../../utils/variables/variables';
 @use 'variables.stack-view' as stack-view-variables;
-
-@mixin lastElementBottomBorder($use-custom-props: true) {
-  border-bottom: none;
-  $fallbackShadow: 0 variables.$clr_baselineRem_1px 0 stack-view-variables.$clr-stack-view-border-box-color;
-  $shadow: 0 variables.$clr_baselineRem_1px 0 var(--clr-stack-view-border-box-color);
-  // IE/OldEdge
-  box-shadow: $fallbackShadow;
-  @if $use-custom-props == true {
-    box-shadow: $shadow;
-  }
-}
+@use '@cds/core/tokens/tokens.scss';
 
 @include meta.load-css('properties.stack-view');
 
 @include mixins.exports('stack-view.clarity') {
   .stack-header {
-    font-weight: stack-view-variables.$clr-stack-view-font-weight;
```

Comment:
Why do we remove this? 
Is the default good enough? NG theme was different?

Reply:
Thanks for the discussion. I agree it's okay to get this removed and documented in the STYLES file.

Reply:
Docs added in commit comment removed in commit be6430d